deploy/crds/hco00.crd.yaml

```diff
@@ -150,6 +150,78 @@ spec:
               bareMetalPlatform:
                 description: BareMetalPlatform indicates whether the infrastructure is baremetal.
                 type: boolean
+              hyperConvergedConfig:
+                description: Let the user define parameters that will be propagated to the CRs of managed components
+                properties:
+                  placement:
+                    description: Placement represent a pod placement configuration that is going to be propagated to other component operators by HCO.
+                    properties:
+                      infra:
+                        description: infra NodePlacement defines the NodePlacement (nodeSelector and tolerations) for all the infra pods managed by operators managed by the HyperConverged Operator. infra generically refers to controllers, apiservers and webhooks; operators are not going to be configured according to this value but only according to spec.config.nodeSelector OLM subscription object
+                        properties:
+                          extraTolerations:
```

Comment:
Does this now mean that operators can freely decide to keep tolerations which they want for their siblings, or do we just mean that some default tolerationons  may be added inn addition? If  the later one, probably sticking with `tollerations` is  better, since then we have the exact pod behaviour.

Reply:
the second one: let's call it `tolerations`.